frontend/app/openproject-app.js

```diff
@@ -44,7 +44,6 @@ require('angular-animate');
 require('angular-aria');
 require('angular-modal');
 
-// require('angular-i18n/angular-locale_en-us');
 if (I18n.locale === 'de') {
   require('angular-i18n/angular-locale_de-de');
```

Comment:
the module is a bit misleading - `angular-i18n` provides localisation as well as internationalisation for different mesages and - most importantly - filters like `currency` and date.

I will investigate further, whether we really need this or if this can be considered cruft. However, this will not be done in the context for this PR :wink: